src/deploy/osp_deployer/checkpoints.py

```diff
@@ -268,7 +349,8 @@ def verify_nodes_registered_in_ironic(self):
         expected_nodes = (len(setts.controller_nodes)
                           + len(setts.compute_nodes)
                           + len(setts.ceph_nodes)
-                          + len(setts.computehci_nodes))
+                          + len(setts.computehci_nodes)
+                          + len(setts.powerflex_nodes))
```

Comment:
Do we differentiate between hci and non-hci? Are we planning to add hci support later?

Reply:
No HCI support is planned as we speack. Nevertheless it's still hard to tell as we don't have any roadmap or requirements...

Reply:
> No HCI support is planned as we speack. Nevertheless it's still hard to tell as we don't have any roadmap or requirements...

ok